sn_client/src/wallet.rs

```diff
@@ -156,12 +207,39 @@ impl WalletClient {
     }
 
     /// Remove CashNote from available_cash_notes
+    //TODO: Unused
     pub fn mark_note_as_spent(&mut self, cash_note_key: UniquePubkey) {
         self.wallet.mark_note_as_spent(cash_note_key);
     }
 
-    /// Send tokens to another wallet.
-    /// Can optionally verify the store has been successful (this will attempt to GET the Spend from the network)
+    /// Send tokens to another wallet. Can also verify the store has been successful.
+    /// This will attempt to GET the Spend from the network.
```

Comment:
The verification is carried out by performing a GET. I think this applies to all the other similar functions with the `verify` option.
```suggestion
    /// Send tokens to another wallet. Can also verify the store has been successful, by
    /// attempting to GET the Spend from the network.
```